crates/sui-graphql-rpc/src/context_data/sui_sdk_data_provider.rs

```diff
@@ -285,14 +285,7 @@ impl DataProvider for SuiClient {
             .map(SerdeBigInt::from);
 
         let pg = self.read_api().get_checkpoints(after, count, false).await?;
-        let system_state = self.governance_api().get_latest_sui_system_state().await?;
-        let protocol_configs = self.fetch_protocol_config(None).await?;
-
-        let data: Result<Vec<_>, _> = pg
-            .data
-            .iter()
-            .map(|c| convert_json_rpc_checkpoint(c, &system_state, &protocol_configs))
-            .collect();
+        let data: Result<Vec<_>, _> = pg.data.iter().map(convert_json_rpc_checkpoint).collect();
```

Comment:
what's the diff between `db_data_provider` and `sui_sdk_data_provider`, are they designed to co-exist?

Reply:
Yes, on setting up the graphql server, one would be able to configure whether the graphql schema is backed by sui-sdk or postgres